app/components/modal/show-request/template.hbs

```diff
@@ -0,0 +1,45 @@
+<Modal-dialog>
```

Comment:
Does it work? components should be called with camel case instead of dash like `<ModalDialog/>`